sclient/README.md

```diff
@@ -28,7 +28,7 @@ curl -H "Host: example.com" http://localhost:3000
 ```
 
 ```html
-<!DOCTYPE html>
+<!doctype html>
```

Comment:
Is this a new convention? I must have missed the change.

Reply:
Says prettier@3.x